content/guides/ethereum/tutorials/track_trx.md

```diff
@@ -31,7 +31,7 @@ then open ({{< external-link href="http://localhost:3000/">}})
 
 ## 3. Add the dfuse Client Library
 
-The simplest way to get started with dfuse and JavaScript/TypeScript development is to use the dfuse JS client library.
+The simplest way to get started with dfuse and JavaScript/TypeScript development is to use the dfuse JS Client Library.
```

Comment:
I’m not sure about this one, I think it was ok before. Ping @joshkauffman for final check.

Reply:
Sure! I saw a change in one of the commits for the title (client library -> Client Library) so I thought you wanted client library to start with cap letters because it was dfuse's Client Library. Either makes sense to me.